apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1009,6 +1009,27 @@ expression: "&schema"
       },
       "additionalProperties": false
     },
+    "experimental_batching": {
+      "description": "Batching configuration.",
+      "default": {
+        "enabled": false,
+        "mode": ""
+      },
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Activates Batching (disabled by default)",
+          "default": false,
+          "type": "boolean"
+        },
+        "mode": {
+          "description": "Batching mode (batch_http_link is supported)",
+          "default": "",
```

Comment:
Should we default to `batch_http_link` mode too?

Reply:
The [specification document](https://docs.google.com/document/d/1CJH17-X9f8htfOOhvGP8t8RwpVu9HvBIFOhagnOmS8Y/edit) specified default to empty.

Reply:
I think the specification doc is wrong. 
The idea is that if we improve batching then users won't suddenly find that their queries are working in a different way.

Reply:
So default to `batch_http_link` mode?

Reply:
I don't have access to that doc, but just from an user perspective there definitely should be an opt-in model with `enable: true` but if there is only one mode we support why not default to that for now?

Reply:
I don't have any feelings about default to None or default to batch_http_link.

@BrynCooke was the original author and (by trawling the spec doc's history) he originally did have it set to default to `batch_http_link` mode. Then, on September 7th, he removed the default and the docs team set it to be `empty`.

Since @BrynCooke was the original author and he now thinks `empty` is wrong, I'm happy to update the default to `batch_http_link`.

Reply:
I've updated the doc. The reason that we want to make mode explicit is that we may want to change the default mode in future in a breaking way.